### PR TITLE
Feature/testing

### PR DESCRIPTION
**Pull request by jasmyn2244**
### What (if any) features are you implementing?
- Api calls are in a new file
### What (if anything) did you refactor?
- removed commented out code
### Were there any issues that arose?
- n/a
### Is there anything that you need from your teammate?
- n/a
### Refactor checklist
- [x] Check variable and parameter names: are they descriptive and unique?
- [x] Is the code DRY? SRP?
- [x] Is it easy for others to read your code? (explanatory comments needed?)
- [x] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [x] Are commit messages formatted correctly?

### CODE REVIEW

File: cypress/integration/main-view.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+describe('Main view page', () => {
```
Comment (anewb87) [inline cypress/integration/main-view.js:1]: cypress testing here we coooome!

File: src/api-calls.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const getAllMovies = () => {
```
Comment (anewb87) [inline src/api-calls.js:1]: Look at us! API calls in their own folder.

**Review by anewb87 — APPROVED** [review]
Organizing, API calls, cleaning up code, starting cypress testing

File: src/App.js [hunk 3]
```diff
@@ -18,43 +19,36 @@ class App extends React.Component {
...
-  // Note: May use this code in the future to check for specific errors
-  // checkError(response) {
-  //   if (response.status === 422 || response.status === 500) {
-  //     // this.setState({error: response.status})
-  //     // this.decideErrorMsg()
-  //   } else {
-  //     return response.json()
-  //   }
+
+  // cleanData = (data) => {
```
Comment (Gabby-Recny) [inline src/App.js:27]: Curious if we could use two cleanData functions for both the Movie Container and Card

**Review by Gabby-Recny — APPROVED** [review]

**Pull request merged**